src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs

```diff
@@ -376,6 +376,29 @@ public async Task PayoutAsync(Balance[] balances)
                 NotifyPayoutSuccess(poolConfig.Id, balances, txHashes.ToArray(), null);
         }
 
+        public decimal getTransactionDeduction(decimal amount)
+        {
+            decimal deduct = 0;
+            if(extraConfig?.MinersPayTxFees == true)
+            {
+                 // the gas limit of a single address->address transaction is currently fixed at 21000
+                var gasAmount = extraConfig?.Gas ?? 21000;
+
+                 // if no MaxGasLimit is configured, nothing will be deducted.
+                var gasPrice = extraConfig?.MaxGasLimit  ?? 0;
```

Comment:
Just to remind we have two max gas limits 60Gwei and 50Gwei depend on the payout time. We are deducting the maximum now in all cases.

Reply:
checked with Mike.  MaxGasLimit is the one to use.